cpp/mysql_list_tables.cc

```diff
@@ -50,6 +50,15 @@ void ProcessTablesOrViews(DbConnection * const db_connection, const bool process
 }
 
 
+void ProcessTriggers(DbConnection * const db_connection, const std::string &database_name) {
+    db_connection->queryOrDie("SHOW TRIGGERS FROM `" + database_name + "`");
+    DbResultSet result_set(db_connection->getLastResultSet());
+    while (const auto row = result_set.getNextRow())
+        std::cout << "CREATE TRIGGER `" << row["Trigger"] << "` ON `" << database_name << '.' << row["Table"]
```

Comment:
This does not look like the official CREATE TRIGGER syntax:
https://dev.mysql.com/doc/refman/8.0/en/create-trigger.html

- How about using the original namings for the parts? e.g. `body` instead of `statement`
- `trigger_time` and `trigger_event` must occur before the `ON` keyword
- what about `trigger_order`?
- `` `database_name.table_name` `` could be invalid, it should be `` `database_name`.`table_name` `` instead

Reply:
I am not sure, what you mean here.  The "names" are what mysql is using for the columns.  I can't use anything else.